libs/interfaces/src/lib/chains.ts

```diff
@@ -1,11 +1,13 @@
 import { PublicKey, SecretKey, KeyPair, secp256k1, ed25519 } from './crypto';
-import { Pair } from '@baf-wallet/utils';
 import { Account as NearAccount } from 'near-api-js';
 import { GenericTxAction, GenericTxParams } from './tx';
 import { Env } from './configs';
+import { Pair } from './misc';
 
+export type ChainAsType = 'near' | 'ethereum';
```

Comment:
What exactly does this do? Is this for code I wrote a long time ago that expected it to be a type and not a enum?

Reply:
This is just to make the api easier for the library. Like you can do tokenInfo('near') instead of Chains.Near